src/Selenium/Monad.purs

```diff
@@ -77,6 +79,15 @@ wait :: forall e o. Selenium e o Boolean -> Int -> Selenium e o Unit
 wait check time = ReaderT \r ->
   S.wait (runReaderT check r) time r.driver
 
+-- | Tries the provided Selenium computation repeatedly until the provided timeout expires
```

Comment:
I haven't included this in `Selenium.purs` because there `Control.Monad.Aff.Reattempt.reattempt` can be used directly.